pysat/_instrument.py

```diff
@@ -309,6 +313,56 @@ def __init__(self, platform=None, name=None, tag=None, inst_id=None,
             estr = 'pad must be a dictionary or a pandas.DateOffset instance.'
             raise ValueError(estr)
 
+        # Store kwargs, passed to standard routines first
+        self.kwargs = {}
+        saved_keys = []
+        partial_func = ['_list_files_rtn', '_download_rtn', '_default_rtn',
+                        '_clean_rtn']
+        for fkey in ['_list_files_rtn', '_load_rtn', '_default_rtn',
+                     '_download_rtn', '_list_remote_files_rtn', '_clean_rtn']:
+            func = getattr(self, fkey)
+
+            # first, check if keywords are  valid
+            good_kwargs = _check_if_keywords_supported(func, **kwargs)
+
+            # get and apply default values for custom keywords
+            default_keywords = _get_supported_keywords(func)
+
+            # store user supplied keywords
+            self.kwargs[fkey] = {gkey: kwargs[gkey] for gkey in good_kwargs}
+
+            # Add in defaults if not already present
```

Comment:
Wait... that won't matter.  If `kw_one="hi"` in load and `kw_one="sad"` in download, it won't cross-contaminate since the kwargs are stored by routine.